src/components/onboarding.tsx

```diff
@@ -77,17 +77,15 @@ export default function Onboarding() {
 
 	useEffect( () => {
 		const run = async () => {
-			const { path, name, isWordPress } = await getIpcApi().generateProposedSitePath(
-				generateSiteName( [] )
-			);
+			const siteName = await generateSiteName( [] );
+			const { path, name, isWordPress } = await getIpcApi().generateProposedSitePath( siteName );
 			setSiteName( name );
 			setProposedSitePath( path );
 			setSitePath( '' );
 			setError( '' );
 			setDoesPathContainWordPress( isWordPress );
 		};
 		run();
-		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Let's bring this line back to signal that we are intentionally excluding those effect dependecies.